src/types/StatusFilterType.ts

```diff
@@ -0,0 +1,5 @@
+export enum StatusFilterType {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
```

Comment:
Why is it uppercase?

Reply:
The use of uppercase letters in the enum values is a convention typically followed in programming languages. It helps to distinguish enum values from regular variables and constants. By using uppercase letters, it provides a visual cue that these values represent predefined constants rather than regular variables that can be changed.